pom.xml

```diff
@@ -11,23 +11,60 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
I have similar modifications in my branch. This means that we will have a conflict once we merge both branches on the master